### DISCUSSION TITLE
Multiple session

### DISCUSSION THREAD

**Discussion by agusibrahim** [dimaskiddo/go-whatsapp-multidevice-rest#40] [Ideas]
halo gan, bisa dibuatkan multi session?

**Comment by hrizal** [comment]
Multi session teh maksudnya gimanah ?

Pada tanggal Sel, 11 Jul 2023 20.52, Agus Ibrahim ***@***.***>
menulis:

**Comment by ibnux** [comment]
Dari judul repo sudah multi device

1 aplikasi ini bisa untuk banyak nomor

Saya pake juga di sini wa.nux.my.id

**Comment by agusibrahim** [comment]
maksudnya multisession bisa scan/login dari bnyk no hp yg berbeda. jd pas kirim pesan bisa milih pake sesi/nohp mana

**Reply by ibnux** [reply]
sudah bisa dari awal repo ini ada   
Waktu auth pakai Nomor HP yang didaftarkan   
https://github.com/dimaskiddo/go-whatsapp-multidevice-rest/wiki/Endpoint#auth   
Tokennya simpan dengan nomor HPnya   
bisa teks file, nama file nomor HP, isinya token   

Script disini hanya gateway, urusan bisa broadcast, pilih nomor HP urusan penggunanya dengan membuat script/aplikasinya sendiri

**Reply by agusibrahim** [reply]
oh pake auth yg berbeda buat bedain no hp. siap saya coba

**Comment by hrizal** [comment]
seharusnya yang dibuat mas Dimas ini memang untuk tujuan itu :)

Pada tanggal Rab, 12 Jul 2023 pukul 08.04 Agus Ibrahim <
***@***.***> menulis:

**Reply by agusibrahim** [reply]
iya gan makasih

**Comment by dimaskiddo** [comment]
Sesuai yang di infokan pak @ibnux dan pak @hrizal ya mas @agusibrahim bahwa memang sudah support multi session setiap session connection dipisah berdasarkan auth nya, dimana secara under the hood auth akan merujuk ke mapping dari masing-masing client connectionnya

Semoga membantu ya 😄
